Literature/e-Books.md

```diff
@@ -33,7 +33,7 @@ description: Reliable sources to obtain e-Books.
 # Scientific Journals/Technical Manuals
 
 - [**Sci-Hub (.se)**](https://sci-hub.se/) | [**.st**](https://sci-hub.st/) | [**.ru**](https://sci-hub.ru/)    
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Sci-Hub X Now!](https://chrome.google.com/webstore/detail/sci-hub-x-now/gmmnidkpkgiohfdoenhpghbilmeeagjj) (Chromium) - Opens the sci-hub page for the article you want to read.  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[Sci-Hub X Now!](https://github.com/gchenfc/sci-hub-now) [Github]
```

Comment:
any specific reason you removed the description?

Reply:
if by description you mean [Github], it was because of the comment above.

Reply:
you can keep the extension description but remove [Github].

Reply:
Oh i completelly missed the fact that i removed that aaa
just used vim's d$ without thinking :skull: